uber/site_sections/registration.py

```diff
@@ -878,7 +874,7 @@ def arbitrary_charge(self, session, payment_id, stripeToken):
                 what=charge.description,
                 reg_station=cherrypy.session.get('reg_station')
             ))
-            raise HTTPRedirect('arbitrary_charge_form?message={}', 'Charge successfully processed')
```

Comment:
We used this page as a way for admins to process sales via Stripe instead of Square. I'll probably have to change this around a bit to get back the old charge form functionality.

Reply:
Due to the vagaries of our Stripe forms, getting this to return to the correct page based on where you accessed this form is kind of sort of impossible. This will have to wait until I have time to do a full refactor.